### PR TITLE
feat: update for react native v7

### PR DESCRIPTION
**Pull request by dannyhw**
# about

- Updates mentions of storybook.requires.js to storybook.requires
- Slightly adjusted explanation of how generate now works
- template project was updated separately https://github.com/chromaui/intro-storybook-react-native-template/pull/4

### CODE REVIEW

File: content/intro-to-storybook/react-native/en/simple-component.md [hunk 1]
```diff
@@ -35,7 +35,9 @@ module.exports = {
...
+In React Native Storybook we use the config in `main.js` to generate a file called `storybook.requires`, this is due to some limitations with the metro bundler. This file gets generated when you run `yarn storybook` to start Storybook or `yarn storybook-generate` which just regenerates the `storybook.requires` file. This file is used to load all the stories in the project and it also imports your addons, whenever you find that a story is not being loaded you can try regenerating this file. Since v7 we are now able to dynamically import new stories that match your config so you shouldn't need to regenerate this file unless you change your `main.js` config.
+
+You'll also find the generate function in metro.config.js that will run the generation whenever you start metro.
```
Comment (shilman) [inline content/intro-to-storybook/react-native/en/simple-component.md:40]:
```suggestion
You'll also find the generate function in `metro.config.js` that will run the generation whenever you start metro.
```

**Review by shilman — APPROVED** [review]
Great job @dannyhw !! 🚀

File: content/intro-to-storybook/react-native/en/simple-component.md [hunk 2]
```diff
@@ -35,7 +35,9 @@ module.exports = {
...
+In React Native Storybook we use the config in `main.js` to generate a file called `storybook.requires`, this is due to some limitations with the metro bundler. This file gets generated when you run `yarn storybook` to start Storybook or `yarn storybook-generate` which just regenerates the `storybook.requires` file. This file is used to load all the stories in the project and it also imports your addons, whenever you find that a story is not being loaded you can try regenerating this file. Since v7 we are now able to dynamically import new stories that match your config so you shouldn't need to regenerate this file unless you change your `main.js` config.
+
+You'll also find the generate function in `metro.config.js` that will run the generation whenever you start metro.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/react-native/en/simple-component.md:40]: @dannyhw we have a bit of room for improvement here. What do you think about rephrasing this as follows:

```suggestion
In Storybook for React Native, due to current limitations with the Metro bundler, we rely on the configuration in `main.js` to generate a file called `storybook.requires`, which is used to load all our stories and addons in our project. This file is automatically generated when you run `yarn storybook` to start your Storybook.

<div class="aside">

💡 You can also manually generate the `storybook.requires` file by running `yarn storybook-generate`. However, you shouldn't need to re-create this file unless you see that a story is not being loaded or notice that a change in your `main.js` config is not reflected in your Storybook. To learn more about how the `storybook.requires` file is generated, you can check out the `generate` function in your `metro.config.js` file.

</div>
```

With it, we can share the same information without going too deep into the specifics nd keep the momentum going in terms of flow.

File: content/intro-to-storybook/react-native/en/composite-component.md [hunk 3]
```diff
@@ -135,7 +135,7 @@ export const Empty = {
 
 `TaskStory.args.task` supplies the shape of a `Task` that we created and exported from the `Task.stories.js` file. Similarly, the `argTypes` we added for `onPinTask` and `onArchiveTask` tell Storybook to provide actions (mocked callbacks) that the `TaskList` component needs.
 
-Since we've added a new story file, we need to run `yarn storybook-generate` again to regenerate the `storybook.require.js` file.
+If you don't immediately see the new story show up then try reloading the app. If that doesn't work you can run `yarn storybook-generate` again to regenerate the `storybook.requires` file.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/react-native/en/composite-component.md:138]: @dannyhw, when you have a moment, can you rephrase this to:
```suggestion
If you don't see the new story immediately, try reloading the app. If that doesn't work, you can re-run `yarn storybook-generate` to regenerate the `storybook.requires` file.
```

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@dannyhw appreciate you taking the time to put together this pull request and update the tutorial for the newly released version 🙏 ! I left a couple of suggestions for you to look into when you have a moment. 

Looking forward to hearing from you.

Have a great week.


Stay safe.

**Review by jonniebigodes — APPROVED** [review]

**Comment by jonniebigodes** [comment]
@dannyhw appreciate you addressing the feedback so promptly. All is good on my end. Going to go ahead and merge it.

**Comment by dannyhw** [comment]
@jonniebigodes thanks! I merged the template changes too 👍

**Comment by jonniebigodes** [comment]
That's fantastic to hear. Appreciate you getting the template updated as well 🙇
